src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -31,6 +34,27 @@ public ConfigurationRepository(Clients clients) {
     return lookupConfigurations(cqlQueryResult, applySearchSchedulerNoticesLimit());
   }
 
+  /**
+   * Gets loan history tenant configuration - settings for loan anonymization
+   *
+   */
+  public CompletableFuture<Result<LoanHistorySettings>> loanHistoryConfiguration() {
+    return defineModuleNameAndConfigNameFilter("LOAN_HISTORY", "loan_history")
```

Comment:
I updated the description of the pull request.

Reply:
Why was the description of the pull request changed?

I'm asking the question because `loan history settings` does not mean much to me. It doesn't reveal the context the settings are used. If they are configuration for anonymization, could this be renamed to describe that use.

Reply:
The class has been renamed.

Reply:
Please can the method in the configuration repository also be renamed as well